simulation/traffic_simulator/include/traffic_simulator/entity/vehicle_entity.hpp

```diff
@@ -37,9 +37,27 @@ namespace entity
 class VehicleEntity : public EntityBase
 {
 public:
-  VehicleEntity(
-    const std::string & name, const traffic_simulator_msgs::msg::VehicleParameters & parameters,
-    const std::string & plugin_name = "behavior_tree_plugin/VehicleBehaviorTree");
+  struct BuiltinBehavior
+  {
+    static auto behavior_tree() noexcept -> const std::string &
+    {
+      static const std::string name = "behavior_tree_plugin/VehicleBehaviorTree";
+      return name;
+    }
+
+    static auto context_gamma() noexcept -> const std::string &
+    {
+      static const std::string name = "TODO";
+      return name;
+    }
+
+    static auto default_behavior() noexcept -> const std::string & { return behavior_tree(); }
+  };
+
+  explicit VehicleEntity(
+    const std::string & name,                                //
+    const traffic_simulator_msgs::msg::VehicleParameters &,  //
+    const std::string & = BuiltinBehavior::default_behavior());
```

Comment:
Can you follow section 4.4 and 4.5 in this coding guidelines?

http://wiki.ros.org/CppStyleGuide

Reply:
1bd92c2 fixed this.